### PR TITLE
ICRC-39: extract icrc-39_batch_calls

### PR DESCRIPTION
**Pull request by plitzenberger**
This PR intends to extract the following standard extensions:

- ICRC-39: Batch Calling

This is a collaborative change by @frederikrothenberger and @plitzenberger

### CODE REVIEW

File: topics/icrc_34_batch_calling.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    * [Message Processing](#message-processing)
+    * [Errors](#errors)
+
+## Summary
+
+JSON-RPC defines a [batch call](https://www.jsonrpc.org/specification#batch) as a JSON array of requests. All methods defined in this standard may also be invoked as part of a batch.
+
+If a signer receives a batch call, it must process each request sequentially in order of the id and reply with a batch response. Calls resulting in error responses do not prevent the processing of subsequent calls in the batch.
+
+## Method
```
[the remark this answers is not in the document]
Comment (plitzenberger) [reply topics/icrc_34_batch_calling.md:25]: agreed

**Comment by frederikrothenberger** [comment]
@plitzenberger : This PR now needs updating with a different number (because of #116 ).

**Comment by frederikrothenberger** [comment]
> @plitzenberger : This PR now needs updating with a different number (because of #116 ).

I assigned it ICRC-39.

**Comment by frederikrothenberger** [comment]
@plitzenberger: I changed the processing of additional requests after a failure to avoid footguns resulting from partially successful batches.

**Review by plitzenberger — COMMENTED** [review]
Looks good to me. But I cannot approve as I'm the original author of the PR.

**Review by frederikrothenberger — APPROVED** [review]

**Pull request merged**